packages/venia-ui/lib/components/FilterModal/filterSearch.js

```diff
@@ -1,41 +0,0 @@
-import React from 'react';
```

Comment:
Was surprised to see entire file deletions like this - was this just old, unused code?

Reply:
Yeah, old unused code. I checked with the team in the slack channel before deleting this to be sure.